tasks/build/browserify.js

```diff
@@ -45,6 +52,10 @@ module.exports = function(grunt) {
           .ignore('i18next-browser-languagedetector');
       }
 
+      if (!isDev) {
+        browseified = browseified.exclude('../../translations/dev');
```

Comment:
This is nit-picky as it wasn't you that first wrote the variable but since you are interacting with it can you change this variable to `browserified`? The missing r in browser feels like a typo.

Reply:
Yes, that stuck out a bit for me too. I had initially assumed that it might be some accepted nomenclature for some reason, but seems like this is [the only repo](https://www.google.com/search?q=%22browseified%22) that uses it. So it must be a typo. I'll change that to `browserify`